docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,97 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
+* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"}
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function uses the Unix Epoch to calculate a date-time equivalent of a specified integer, then returns the results.
+
+## Syntax
+
+```
+TOTIMESTAMP([int_val | expr], timeunit)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| int_val | integer | Integer value to be converted to a timestamp | Yes | |
+| expr | | expression used to derive an integer value to be converted to a timestamp | Optional | |
+| timeunit | string | String value that specifies the unit of time to convert. | Optional | [Time units](#additional-information).|
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `timestamp` | Returns timestamp equivalent of `int_exp`, it converts the input using unix epoch as base and the optional `timeunit` to determine the granularity of the input value. |
+
+## Additional information
+
+### `timeunit`
+
+`timeunit` defaults to `s` if not specified.
+
+{% include /sql-guide/timestamp-timeunit-table.md %}
+
+## Examples
```

Comment:
Is there an example we can show that's not an insert? Does this work as in a SELECT when pulling an int column? I realize it wasn't the intent of this functionality, but assuming it can be done?